ibis/impala/tests/common.py

```diff
@@ -23,6 +23,12 @@
 import ibis.compat as compat
 import ibis
 
+try:
+    import impyla
```

Comment:
What you had before using `importorskip` is fine, you just need to change `impyla` to `impala`.